test/e2e_node/container_lifecycle_test.go

```diff
@@ -236,7 +236,7 @@ var _ = SIGDescribe("[NodeConformance] Containers Lifecycle ", func() {
 						Name:  regular1,
 						Image: busyboxImage,
 						Command: ExecCommand(regular1, execCommand{
-							Delay:    2,
+							Delay:    10,
```

Comment:
pretty much LGTM, but there are 2 questions:

1. how was "10" determined? If it was done by trial and error, fine, but please let's record that - so if and when we revisit in the future we have a data point to guide our future decisions. If a computation was made, or anything, even better, let's just note it down
2. do we know, or can we research (git history?) why "2" was selected in the first place?

Reply:
1. looking at the error message, it seems that there can be 1~2 seconds delay before the container start its postStart. commented it for the future reader.
2. not sure about the original decision but I guess we didn't expect it to flake. https://github.com/kubernetes/kubernetes/pull/116426 @SergeyKanzhelev

Reply:
thanks @gjkim42 from my POV both additions answers my questions.